helm/charts/text-generation-inference/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
```

Comment:
helm lint advise to add an icon, if you have one for TGI I can amend the PR do add it.